nixos/tests/installer.nix

```diff
@@ -71,20 +81,24 @@ let
   # partitions and filesystems.
   testScriptFun = { bootLoader, createPartitions, grubDevice, grubUseEfi, grubIdentifier
                   , postInstallCommands, preBootCommands, postBootCommands, extraConfig
-                  , testSpecialisationConfig, testFlakeSwitch
+                  , testSpecialisationConfig, testFlakeSwitch, clevisTest, clevisFallbackTest
                   }:
     let iface = "virtio";
         isEfi = bootLoader == "systemd-boot" || (bootLoader == "grub" && grubUseEfi);
         bios  = if pkgs.stdenv.isAarch64 then "QEMU_EFI.fd" else "OVMF.fd";
     in if !isEfi && !pkgs.stdenv.hostPlatform.isx86 then ''
       machine.succeed("true")
     '' else ''
+      import subprocess
+      tpm_folder = os.environ['NIX_BUILD_TOP']
```

Comment:
nit/future: in the future, this has to be cleaned up to use the TPM infrastructure rather than a bespoke system and we should fix whatever doesn't work for you later on.

Reply:
I think the issue there is that the installer tests have a whole custom qemu invocation for the installed system, so all of the `qemu-vm.nix` options simply don't apply.

Reply:
Also, I think the `swtpm` socket is killed when the vm reboots, which it does multiple times in these tests.